CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Changes
+- Fixed 'Raise helpful error if the same needs is not unique[#531](https://github.com/luckyframework/lucky/issues/531)
+
```

Comment:
Could you remove this, we update this file ourselves before a release

Reply:
Hi Matthew - I've removed the Change Log comment.